src/BloomFilter/Persist/BitRedis.php

```diff
@@ -68,7 +68,7 @@ public function getBulk(array $bits): array
             $pipe->getBit($this->key, $bit);
         }
 
-        return $pipe->exec();
+        return is_array($return = $pipe->exec()) ? $return : [$return];
```

Comment:
Just matter of taste and for me clean code. Assign the value to `$return` outside of `is_array` function

Reply:
Fixed.